seqr/utils/elasticsearch/es_search.py

```diff
@@ -1037,7 +1037,7 @@ def _liftover_grch38_to_grch37():
         try:
             LIFTOVER_GRCH38_TO_GRCH37 = LiftOver('hg38', 'hg19')
         except Exception as e:
-            logger.warn('WARNING: Unable to set up liftover. {}'.format(e))
+            logger.error('WARNING: Unable to set up liftover. {}'.format(e))
```

Comment:
Using `logger.error()` but there is `WARNING` in the message. Should the `WARNING` be changed to `ERROR`?

Reply:
done